tests/lib/cmsis_nn/src/main.c

```diff
@@ -146,7 +146,7 @@ const q7_t conv_4_output_ref[36] = {
 	4, 5, 14, 2, 5, 7, -1, -2, 2, 5, -4, 11, -1, -2, 8, 4, 2, 0
 };
 
-void test_convolve(void)
+ZTEST(test_cmsis_nn, test_convolve)
```

Comment:
remove leading test_ from testsuite name.

Reply:
Removed test_ for testsuite name.